CRM/Upgrade/Incremental/php/FiveFortySeven.php

```diff
@@ -69,10 +69,6 @@ public function upgrade_5_47_alpha1($rev): void {
       "timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP COMMENT 'Relationship last modified.'"
     );
     $this->addTask('Set initial value for relationship created_date and modified_date to start_date', 'updateRelationshipDates');
-    $this->addTask('core-issue#2122 - Add timezone column to Events', 'addColumn',
```

Comment:
I didn't notice before that this duplicates the same removal in #22930, but I guess it works out the same in the end.